modules/dbSchema/user.js

```diff
@@ -0,0 +1,23 @@
+var mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+var fs = require('fs');
+var logger = require('.././logger').logger;
+
+var UserSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        unique: true,
+        required: true
+    },
+    password: {
+        type: String,
+        required: true
+    }
+});
+UserSchema.methods.validPassword = function( pwd ) {
+    return ( this.password === pwd );
```

Comment:
this is thing, that i want to do (and passport checking for all requests) but dont do that yet